apps/prairielearn/src/lib/externalGradingSocket.js

```diff
@@ -32,7 +33,14 @@ module.exports.connection = function (socket) {
     socket.join(`variant-${msg.variant_id}`);
 
     module.exports.getVariantSubmissionsStatus(msg.variant_id, (err, submissions) => {
-      if (ERR(err, (err) => logger.error('Error getting variant submissions status', err))) return;
+      if (
+        ERR(err, (err) => {
+          logger.error('Error getting variant submissions status', err);
+          Sentry.captureException(err);
```

Comment:
I followed the procedure used in other parts of the code regarding Sentry, I hope this is correct. I don't know how to test this. To confirm, would this require additional initialization as well, since it's not part of a "normal request" flow? I see that cron doesn't require the init, so I assume this one doesn't either.

Reply:
Yep, this is all you have to do - Sentry uses a global singleton-ish instance.